packages/engine/Source/Scene/Cesium3DTileset.js

```diff
@@ -2694,8 +2753,9 @@ function updateTiles(tileset, frameState, passOptions) {
     statistics.incrementSelectionCounts(tile.content);
     ++statistics.selected;
   }
-  for (i = 0; i < emptyLength; ++i) {
```

Comment:
See "Declare variables where they are first used" in the [coding guide](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md#basic-code-construction)